src/parser/shell_expand/words.rs

```diff
@@ -201,14 +201,14 @@ impl<'a> ArrayMethod<'a> {
             }
         }
 
-       
+
         Some("".into()).into_iter().collect()
     }
 }
 
 #[derive(Debug, PartialEq, Clone)]
 pub enum WordToken<'a> {
-    Normal(&'a str),
+    Normal(&'a str,bool),
```

Comment:
I did it this way to not have an extra if else construct in basically every place that returns a Normal. If you insist I'll gladly change it but I think it's gonna mess up the code.